core/modules/block/block.module

```diff
@@ -8,11 +8,16 @@
  * Implements hook_permission().
  */
 function block_permission() {
-  return array(
+  $permissions = array(
     'administer blocks' => array(
       'title' => t('Administer custom blocks'),
     ),
   );
+  if (module_exists('translation')) {
```

Comment:
I don't think Translation module is needed here, this looks like it should just require `'language'` like everywhere else in this PR.

Reply:
Changed to language